README.md

```diff
@@ -40,24 +40,77 @@ Utilize Plugins to get up and running quickly. Here are some of our top Plugins:
 
  These are many of the Plugins supported by Stanza, with more being developed all the time. View a full list of Plugins [here](https://github.com/observIQ/stanza-plugins/tree/master/plugins).
 
-# Documentation
+# Quick Start
 
-## [Quick Start](./docs/README.md)
-
-### Installation
+## Installation
 
 To install Stanza, we recommend using our single-line installer provided with each release. Stanza will automatically be running as a service upon completion. 
 
-#### Linux/macOS
+### Linux/macOS
 ```shell
 sh -c "$(curl -fsSlL https://github.com/observiq/stanza/releases/latest/download/unix-install.sh)" unix-install.sh
 ```
-#### Windows
+### Windows
 ```pwsh
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; Invoke-Expression ((New-Object net.webclient).DownloadString('https://github.com/observiq/stanza/releases/latest/download/windows-install.ps1')); Log-Agent-Install
 ```
 
-For more details on installation and configuration, check out our full [Quick Start Guide](./docs/README.md)!
+### Kubernetes
+For Kubernetes, there are several guides to install and configure Stanza found [here](./examples/k8s).
+
+## Configuration
+
+To configure Stanza, navigate to the `config.yaml` file located in the Stanza install directory. There are a number of [plugins](./docs/plugins.md) and [operators](./docs/operators/README.md) available to configure in Stanza, but as an example we'll configure a MySQL plugin and a file operator.
```

Comment:
```suggestion
To configure Stanza, navigate to the `config.yaml` file located in the Stanza install directory. There are a number of [plugins](./docs/plugins.md) and [operators](./docs/operators/README.md) available to configure in Stanza, but as an example we will configure a MySQL plugin.
```

- avoid contractions here (And elsewhere in the PR)
- no need to call out the file operator, it is part of the MySQL plugin

Reply:
file operator is called out because it's a separate example used